repose-aggregator/tests/release-verification/src/fake-services/fake-keystone2/package.json

```diff
@@ -1,22 +1,22 @@
 {
-  "name": "ReposeFakeKeystonev2",
-  "version": "0.1.0",
-  "description": "Mock Keystone Service",
-  "readme": "README.md",
+  "name": "service-mocks",
+  "version": "1.0.0",
+  "description": "Simple services that mock the behavior of more complex services like OpenStack Identity.",
```

Comment:
This seems to be a little more generic than we want right now. At least until we move all of them into a single build.